@here/harp-map-theme/resources/berlin_tilezen_base.json

```diff
@@ -1138,7 +1138,7 @@
                     "metalness": 0.8,
                     "emissive": "#78858C",
                     "emissiveIntensity": 0.85,
-                    "vertexColors": true,
+                    "vertexColors": false,
```

Comment:
Can you just make vertexColors=false the default?

Reply:
In fact, they already are. Yeah, I can simply remove that prop.